README.rst

```diff
@@ -166,3 +166,19 @@ where environment variables are the preferred configuration method.
 This module let's me do things the way I prefer in environments I control, but
 still run them with environment variables on environments I don't control with
 minimal fuss.
+
+
+Contribute
+----------
+
+Create virtual environment and install package and dependencies.
+
+.. code:: shell
+    pip install -e .
+    pip install -e ".[tests]"
```

Comment:
You can do this in one go I think... `pip install -e .[tests]`

Reply:
It didn't install the test dependencies running only the first one :thinking:

Reply:
That's correct. You only need the second one. It will install the project _and_ the test dependencies.

Reply:
Sorry! I read it quickly and understood the opposite :facepalm: